docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,532 @@
+    ```
+
+- Time Item:
+
+  This item contains time information.
+    ```
+    ret = noti_ex_item_time_create(&time_item, "time_id", time_info);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+<a name="group"></a>
+## Creating Group Item and add child
+
+- You can add an item as a child of a group using `noti_ex_item_group_add_child()`.
+
+  A group item means a series of items. Viewers can display them in the order they are added child. There can be multiple groups in a notification.
+    ```
+    noti_ex_item_h group_item = NULL;
+    noti_ex_item_h child_item = NULL;
+
+    ret = noti_ex_item_group_create(&group_item, "group_id");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+
+    ret = noti_ex_item_button_create(&child_item, "button_id", "button_title");
+    ret = noti_ex_item_group_add_child(group_item, child_item);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- direction setting
+
+  You can use `noti_ex_item_set_direction()` to decide whether to display the child vertically or horizontally.
+    ```
+    ret = noti_ex_item_group_set_direction(group_item, true);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+<a name="set"></a>
+## Setting Notification-ex Attributes
+
+You can set the following attributes for a notification:
+
+- Id:
+
+  All items are identified by an ID attribute. By default, ID is specified while creating an item but it can also be set separately.
+    ```
+    ret = noti_ex_item_set_id(item_handle, "new_id");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Style:
+
+  Styles with color, padding, and geometry can be added for each item. You can also specify the background image and color.
+
+  - Color
+
+    Color information can be defined as a `notification_ex color` handle with alpha, red, green and blue value.
+    ```
+    noti_ex_color_create(noti_ex_color_h *handle, unsigned char a, unsigned char r, unsigned char g, unsigned char b);
+    ```
+
+  - Padding
+
+    Padding information can be defined as a `notification_ex padding` handle with left, right, top, bottom values.
+    ```
+    noti_ex_padding_create(noti_ex_padding_h *handle, int left, int top, int right, int bottom);
+    ```
+
+  - Geometry
+
+    Geometry information can be defined as a `notification_ex geometry` handle with x, y, width, height values.
+    ```
+    noti_ex_geometry_create(noti_ex_geometry_h *handle, int x, int y, int w, int h);
+    ```
+
+  To set the style attribute, use `noti_ex_item_set_style()`:
+    ```
+    noti_ex_style_create(noti_ex_style_h *handle, noti_ex_color_h color, noti_ex_padding_h padding, noti_ex_geometry_h geometry);
+
+    ret = noti_ex_item_set_style(item_handle, style);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Sound, Vibration and Led:
+
+  Sound and vibration are listed as attributes, hence changing to attribute:
```

Comment:
Please remove this line (280).
"Sound and vibration are listed as attributes, hence changing to attribute:"
This line was my justification for changing the "option" to "attribute".